src/main/java/io/rtr/conduit/amqp/impl/AMQPTransport.java

```diff
@@ -132,6 +139,19 @@ protected String createDynamicQueue(String exchange,
         return queue;
     }
 
+    protected void ensureBasicConfig(String exchange, String exchangeType, String queue, String routingKey, boolean isPoisonQueueEnabled) throws IOException {
```

Comment:
don't think this needs to be protected.  looks like it could be private or even package private if you need visibility for testing.